packages/ckeditor5-engine/src/model/markercollection.js

```diff
@@ -418,6 +429,19 @@ class Marker {
 		return this._affectsData;
 	}
 
+	/**
+	 * Gets the current marker's data.
+	 *
+	 * @returns {module:engine/model/markercollection~MarkerData}
+	 */
+	getData() {
```

Comment:
This method may be enhanced in the future and expose things like marker attributes for the differ.